examples/statemachine/documentSignoffDemo.py

```diff
@@ -4,7 +4,6 @@
 # Example of a state machine modeling the state of a document in a document
 # control system, using named state transitions
 #
-import statemachine
```

Comment:
Even though there are no references to the statemachine module, it must be imported in order for the .pystate modules to be importable.